aws-android-sdk-mobile-client/src/main/java/com/amazonaws/mobile/client/AWSMobileClient.java

```diff
@@ -1301,14 +1302,16 @@ public void confirmSignIn(final Map<String, String> signInChallengeResponse,
      * @throws Exception
      */
     @WorkerThread
-    public SignInResult confirmSignIn(final Map<String, String> signInChallengeResponse) throws Exception {
+    public SignInResult confirmSignIn(final Map<String, String> signInChallengeResponse,
```

Comment:
This would be a breaking change in terms of compilation for a publicly available API. Would prefer to overload this method to accept clientMetadata.

```java
confirmSignIn(signInChallengeResponse);
confirmSignIn(signInChallengeResponse, clientMetadata);
```